tutorials/clinical/10_ieeg_localize.py

```diff
@@ -231,23 +234,21 @@ def plot_overlay(image, compare, title, thresh=None):
 subject_brain = nib.load(
     op.join(misc_path, 'seeg', 'sample_seeg', 'mri', 'brain.mgz'))
 
-# Make brain surface from T1
-verts, triangles = mne.surface.marching_cubes(
-    np.asarray(subject_brain.dataobj), level=100)
-# transform from voxels to surface RAS
-verts = mne.transforms.apply_trans(
-    subject_brain.header.get_vox2ras_tkr(), verts) / 1000.  # to meters
+# get native to head trans
+trans = mne.channels.compute_native_head_t(montage)
+
+# set new montage
+raw.set_montage(montage)
 
+# plot the alignment
 fig_kwargs = dict(size=(800, 600), bgcolor='w', scene=False)
 renderer = mne.viz.backends.renderer.create_3d_figure(**fig_kwargs)
-renderer.mesh(*verts.T, triangles=triangles, color='gray',
-              opacity=0.05, representation='surface')
-for ch_coord in ch_coords:
-    renderer.sphere(center=tuple(ch_coord / 1000.), color='y', scale=0.005)
+fig = mne.viz.plot_alignment(raw.info, trans, 'sample_seeg',
+                             fig=renderer.figure,
+                             subjects_dir=op.join(misc_path, 'seeg'),
+                             show_axes=True, surfaces=dict(pial=0.2))
```

Comment:
Better to have `coord_frame=`mri` here for view comparisons?

Reply:
Not sure I get your meaning. Isn't it in `mri`?

Reply:
default `coord_frame='head'`

https://mne.tools/stable/generated/mne.viz.plot_alignment.html#mne-viz-plot-alignment

Reply:
(you can see in [my comment](https://github.com/mne-tools/mne-python/issues/9520#issuecomment-880968547) that I pass `coord_frame='mri'`, if you don't do this then `plot_alignment` plots everything in head coordinates, while `Brain` always plots in MRI coords)